packages/admin-graphql-api-utilities/src/index.ts

```diff
@@ -1,4 +1,19 @@
-const GID_REGEXP = /\/(\w+(-\w+)*)(?:\?(?:[\w-_]+=[\w-_]+&)*(?:[\w-_]+=[\w-_]+))?$/;
+const GID_TYPE_REGEXP = /^gid:\/\/[\w-]+\/([\w-]+)\//;
+const GID_REGEXP = /\/(\w[\w-]*)(?:\?(.*))*$/;
+
+interface ParsedGID {
```

Comment:
nitpick: We seem to say `Gid` rather than `GID` elsewhere so should this be `ParsedGid`?